web/config/settings.py

```diff
@@ -310,6 +313,16 @@
 SLACK_CLIENT_ID = None
 
 OCTOPRINT_TUNNEL_CAP = int(os.environ.get('OCTOPRINT_TUNNEL_CAP', '1099511627776'))  # 1TB by default
+OCTOPRINT_TUNNEL_SUBDOMAIN_FMT = '{subdomain_code}.tunnels.{site.domain}'
+OCTOPRINT_TUNNEL_SUBDOMAIN_RE = re.compile(r'^(\w+)\.tunnels.*$')
+USE_OCTOPRINT_TUNNEL_V2 = (
```

Comment:
What will be the time when USE_OCTOPRINT_TUNNEL_V2 should be set to False?

Reply:
Probably we can remove this.